charts/generic-govuk-app/templates/worker-deployment.yaml

```diff
@@ -38,41 +38,42 @@ spec:
           {{- . | toYaml | trim | nindent 8 }}
         {{- end }}
       containers:
+        {{- range .Values.workers }}
         - name: app
-          image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
-          imagePullPolicy: {{ .Values.appImage.pullPolicy | default "Always" }}
-          command: ["bundle"]
-          args: ["exec", "sidekiq", "-C", "config/sidekiq.yml"]
+          image: "{{ $.Values.appImage.repository }}:{{ $.Values.appImage.tag }}"
+          imagePullPolicy: {{ $.Values.appImage.pullPolicy | default "Always" }}
+          command: {{ .command | toYaml }}
           ports:
             - name: metrics
-              containerPort: {{ .Values.metricsPort }}
+              containerPort: {{ $.Values.metricsPort }}
           envFrom:
             - configMapRef:
                 name: govuk-apps-env
           env:
-            {{- if .Values.sentry.enabled }}
+            {{- if $.Values.sentry.enabled }}
```

Comment:
Sorry just wondering why there is a $ before calling the values from the values files - just have not seen it being used before?